### DISCUSSION TITLE
🚀 Feature: Standalone-Link zu Alert hinzufügen

### DISCUSSION THREAD

**Discussion by gretaneumannn** [public-ui/kolibri#6611] [Ideas]
### Goals

1. Es soll ein Link mit Höhe von 44px unterhalb des Alert Textes zur Verfügung gestellt werden.
2. Der Link ist optional.

### Non-Goals

1. 
2.
3.

### Background

Für Duplikats-Fehlemeldungen in Formularen, möchten wir innerhalb des Alerts dem Nutzenden einen Link anbieten, damit er direkt zum anderen Datensatz abspringen kann. 

Beispiel:

![image](https://github.com/public-ui/kolibri/assets/118734593/005d9242-6f44-4551-a680-3612bbad239b)

### Proposal

Hierzu bitte bei Fragen auf @Chrisdo82 zugehen.

**Comment by laske185** [comment]
hier sind einige Fragen offen:
- Wie sollten die Links das das KolInputText übergeben werden?
- Wird die Komponente nicht überrachtet? Recht es vielleicht außehalb der Komponente, z.B. darunter, die Liste mit Vorhandenen Objekten darzustellen?

**Comment by deleonio** [comment]
Done: https://github.com/public-ui/kolibri/pull/7879
